src/Sidekick.Business.Tests/ItemParserTests/EquipmentParsing.cs

```diff
@@ -0,0 +1,91 @@
+using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Framework;
+using Sidekick.Business.Apis.Poe.Parser;
+
+namespace Sidekick.Business.Tests.ItemParserTests
+{
+    public class EquipmentParsing : TestContext<ParserService>
+    {
+        [Test]
+        public async Task ParseUnidentifiedUnique()
+        {
+            var actual = await Subject.ParseItem(UnidentifiedUnique);
+
+            using (new AssertionScope())
+            {
+                actual.Type.Should().Be("Jade Hatchet");
+                actual.Identified.Should().BeFalse();
+            }
+        }
+
+        [Test]
+        public async Task ParseSixLinkUnique()
+        {
+            var actual = await Subject.ParseItem(UniqueSixLink);
+
+            using (new AssertionScope())
+            {
+                actual.Name.Should().Be("Carcass Jack");
+                actual.Type.Should().Be("Varnished Coat");
+                actual.Sockets
+                    .Should().HaveCount(6)
+                    .And.OnlyContain(socket => socket.Group == 0);
+            }
+        }
+
+
+        #region ItemText
+
+        private const string UniqueSixLink = @"Rarity: Unique
```

Comment:
@Blinke I think it would be better if these are inline with the test. Uglifies the test but you can see what is being tested.

Reply:
Since they are constants, you can mouse over them in Visual Studio and see the value. I think that's a good balance between cleanliness and readability.